neard/src/runtime.rs

```diff
@@ -113,6 +114,7 @@ impl EpochInfoProvider for SafeEpochManager {
 /// TODO: this possibly should be merged with the runtime cargo or at least reconciled on the interfaces.
 pub struct NightshadeRuntime {
     genesis_config: GenesisConfig,
+    genesis_runtime_config: Arc<RuntimeConfig>,
```

Comment:
why do we need to do this? runtime config is part of `genesis_config`

Reply:
You'd have to clone it every time to wrap into `Arc`

Reply:
It seems that you could implement it with lifetimes, but Arc is fine

Reply:
Lifetimes tend to poison code. It is better if we first solidify our codebase with smart pointers and then switch to references.